src/package/metadata.desktop

```diff
@@ -4,6 +4,7 @@ name[fr]=Dynamique
 Name[nl]=Dynamisch
 Name[ru]=Динамический
 Name[uk]=Динамічний
+Name[it]=Dinamico
```

Comment:
Thanks! Could you please put this entry between `Name[fr]=Dynamique` and `Name[nl]=Dynamisch` so the entries remain sorted alphabetically?

Reply:
Sorry I'm seriously dumb. I'm fixing it